### PR TITLE
Documentation revised

### PR DESCRIPTION
**Pull request by dacappo**
### Description
Description of the target architecture

### Checklist
Before submitting this PR, please make sure:

- [x] Your code builds clean with `make test`
- [x] You have documented new or changed features

### CODE REVIEW

File: docs/devel/architecture.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# karydia Concepts
+## Secure by Default
+The mission of karydia is to configure kubernetes in a more secure way. Many kubernetes default settings are unsecure. When karydia is installed these defaults are revised and supervised. Most karydia features can be switched on centrally and configured on namespace level.
```
Comment (dacappo) [inline docs/devel/architecture.md]: As mentioned by @marwinski, `unsecure` may be an unfavorable wording. E.g. `Many kubernetes default settings are quite permissive`

File: docs/devel/architecture.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# karydia Concepts
+## Secure by Default
+The mission of karydia is to configure kubernetes in a more secure way. Many kubernetes default settings are unsecure. When karydia is installed these defaults are revised and supervised. Most karydia features can be switched on centrally and configured on namespace level.  
+
+## Configuration
+karydia provides the possibility to select the security features which should be used and supervised in a kubernetes cluster (deployment.yml). More specific settings can be made in a [KarydiaSecurityPolicy](/manifests/../../example-karydia-security-policy.yml). If these settings shall be adapted for a selected namespace it is possible to overwrite it with an annotation.
```
Comment (dacappo) [inline docs/devel/architecture.md]: Referring to settings: ... to overwrite **them** with annotations.

File: docs/devel/architecture.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+### Controller
+The karydia policy controller will checks regularly for new or modified artifacts (e.g. namespaces). If there are relevant changes, karydia will add corresponding security settings. This means for example: If a namespace is created karydia will create a default network policy for the new namespace.
+
+![Installing karydia](../images/CreateNamespace.png)
+
+
+## Reconcile
+karydia security settings can be disabled by an administrator. The karydia reconciling should guarantee that any activated security setting is forced, even so it is deleted manually. This means if an administrator deletes a karydia network policy it will be recreated by karydias reconciler.
```
Comment (dacappo) [inline docs/devel/architecture.md]: The reconciling guarantees that the default security setting is forced. I assume deleting an annotation that marks an exception from the default would lead to a fallback to the default. Even for an exceptional network policy annotated on a namespace. We simply cannot store the exceptions, because we trust the changed annotation. 

TL;DR; We only guarantee what is annotated?

**Comment by SAPJonasZengerle** [comment]
@CodeClinch & @dacappo: What's the current status here? Should we proceed with the process or can we reject and delete this PR?

**Comment by SAPJonasZengerle** [comment]
closing PR because of inactivity
